src/background.ts

```diff
@@ -138,6 +139,22 @@ protocol.registerSchemesAsPrivileged([
 
 const firstUrl = process.env.VITE_DEV_SERVER_URL ?? "app://./index.html";
 
+const tempProjectPath = path.join(fixedUserDataDir, "project.tmp.json");
```

Comment:
ここももう「プロジェクト」じゃないかなと！
こんな感じ？
```suggestion
const tempWorkspacePath = path.join(fixedUserDataDir, "workspace.tmp.json");
```

このあたり、`temp`なくても意味通じそうかもとちょっと思いました。

Reply:
すみません、命名周りは後で修正しようと思ってまだできていませんでした...
Electron側の修正は完了しましたが、まだ `project.ts` のアクション名やコメントなどでいくつか修正箇所がありそうなので追って対応します。